### ISSUE TITLE
AttributeError: module 'imgcat' has no attribute 'FigureCanvas'

### ISSUE THREAD

**Issue by toyama0919** [wookayin/python-imgcat#19]
I got the following error.
It seems that `matplotlib.use(module://imgcat)` is not working.

Below is an example of an error made by a one-liner
I get the same error with `matplotlib.use(module://imgcat)`

```
$ MPLBACKEND="module://imgcat" python -c 'import imgcat, matplotlib.pyplot as plt; plt.show()'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/hiroshi.toyama/.pyenv/versions/3.10.8/lib/python3.10/site-packages/matplotlib/pyplot.py", line 526, in show
    _warn_if_gui_out_of_main_thread()
  File "/Users/hiroshi.toyama/.pyenv/versions/3.10.8/lib/python3.10/site-packages/matplotlib/pyplot.py", line 441, in _warn_if_gui_out_of_main_thread
    canvas_class = cast(type[FigureCanvasBase], _get_backend_mod().FigureCanvas)
  File "/Users/hiroshi.toyama/.pyenv/versions/3.10.8/lib/python3.10/site-packages/matplotlib/pyplot.py", line 280, in _get_backend_mod
    switch_backend(rcParams._get("backend"))  # type: ignore[attr-defined]
  File "/Users/hiroshi.toyama/.pyenv/versions/3.10.8/lib/python3.10/site-packages/matplotlib/pyplot.py", line 343, in switch_backend
    canvas_class = module.FigureCanvas
AttributeError: module 'imgcat' has no attribute 'FigureCanvas'
```

**Comment by wookayin** [comment]
I think this is already fixed by #13. Can you try the master version?

**Comment by wookayin** [comment]
Tested with matplotlib 3.7.2. It was broken on 0.5.0 with recent matplotlib. Closing as a duplicate.

<img width="631" alt="image" src="https://github.com/wookayin/python-imgcat/assets/1009873/aeb9dd95-eb4a-47df-8299-331fb0544a76">
